mwdb/resources/metakey.py

```diff
@@ -394,6 +398,82 @@ def put(self, key):
         schema = MetakeyDefinitionManageItemResponseSchema()
```

Comment:
Maybe this should return MetakeyDefinitionItemResponseSchema as well :thinking:

Reply:
Yes It should, because we don't return attribute permissions here.